cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
```

Comment:
Basically we want to print `127.xxx.xxx.xxx` at the end of list.  I would prefer custom sorting by checking `strings.HasPrefix(IPStr, "127.")` .  Number sorting would lead to failure like the one you mentioned below.